packages/docs/docs/provider/index.md

```diff
@@ -4,34 +4,55 @@ sidebar_position: 1
 
 # Medplum Provider
 
-The Medplum Provider is a user-facing web application available at [https://provider.medplum.com/](https://provider.medplum.com/) targeted towards clinical users and operations teams.  Medplum Provider is built on the Medplum Platform and is [open source](https://github.com/medplum/medplum/tree/main/examples/medplum-provider).
+The Medplum Provider is a user-facing web application for clinical users and operations teams. In this section, you can learn about different features of the Provider App and the workflows they enable.
 
-You can use the Medplum Provider as a starting point or reference implementation for your custom electronic health record system.
+Medplum Provider is built on the Medplum Platform and is [open source](https://github.com/medplum/medplum/tree/main/examples/medplum-provider). Depending on your organization’s needs, you can use Medplum Provider as your system of record, as a starting point, or as an implementation reference for your own custom electronic health record system.
 
-:::warning 🚧 Under Construction
-Medplum provider is in active development, check back frequently for updates.
+<br />
+
+:::warning Under Construction
+Medplum Provider is in active development—please check back frequently for updates.
 :::
 
+<br />
+
 ## Getting Started
 
-To use Medplum Provider, you'll need a Medplum account
+To use Medplum Provider, you'll need a Medplum account.
+
+1. Register a new account and Project on the Medplum App at [app.medplum.com](http://app.medplum.com)  
+2. Navigate to Medplum Provider at [provider.medplum.com](http://provider.medplum.com) and sign in with the same credentials
+
+Occasionally, you may also need to use the Medplum App for administrative and other tasks which we note explicitly in this documentation.
+
+<br />
+
+## Using Medplum Provider
+
```

Comment:
Maybe add some text here - something like: 

The following sections outline the primary functionality of Medplum provider or similar for context on what this section is.